ci/tasks/send-hg-metric.yml

```diff
@@ -0,0 +1,26 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+    tag: 3.12.0
+
+inputs:
+  - name: src
+  - name: release-info
+
+params:
+  METRIC_NAME: metricname
+  METRIC_VALUE: metricvalue
+
+run:
+  path: bash
+  dir: src
+  args:
+    - -eu
+    - |
+        apk add netcat-openbsd
```

Comment:
Does concourse re-use this container or is it installing netcat-openbsd every single time we send a metric? If so we might want to create an image with it already on and use that?